src/excalidraw-utils/README.md

```diff
@@ -0,0 +1,70 @@
+# @excalidraw/utils
+
+## Install
+
+    npm i @excalidraw/utils
+
+## API
+
+### `exportToBlob` (async)
+
+Export an Excalidraw diagram to a [Blob](https://developer.mozilla.org/en-US/docs/Web/API/Blob).
+
+### `exportToSvg`
+
+Export an Excalidraw diagram to a [SVGElement](https://developer.mozilla.org/en-US/docs/Web/API/SVGElement).
+
+## Usage
+
+In a Web page:
```

Comment:
we should add the usage when imported as a module as well

Reply:
Speaking of module, it might be nice to publish it as an ES6 module (in addition to UMD). It would allow to use it as follow (without using a bundler):

```html
<script type="module">
  import { exportToSvg } from './node_modules/@excalidraw/utils/dist/excalidraw-utils.mjs'

  const excalidrawDiagram = {}
  exportToSvg(excalidrawDiagram)
</script>
```